orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/rollback/PreviousImageRollbackSupport.groovy

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
No javas?

Reply:
That’s fair, this was originally just lifted from an existing  groovy class but I can add a bunch more null checks and do the java conversion.